transcom/mymove
# INT-B-19226 Revert and redo of form changes

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19226)

## Relevant INT PRs
#13474
#13548 
#13567

## Summary

This PR changes customer onboarding and move creation forms in MilMove to label required fields with light grey "Required" text, while getting rid of the existing "Optional" text that currently accompanies optional fields. While this was handled with the other INT PRs, the method used changed behavior across the entire app, which was not in the story scope. Now only forms on the Customer side should be affected.


### How to test
Please follow the steps in the relevant INT PRs, but basically, check all customer side onboarding and move forms to check that all form fields that are needed to submit show a "Required" light grey help label, and that no "Optional" labels are left over.
